docs/sources/http_api/short_url.md

```diff
@@ -0,0 +1,77 @@
++++
+title = "ShortURL HTTP API "
+description = "Grafana ShortURL HTTP API"
+keywords = ["grafana", "http", "documentation", "api", "shortUrl"]
+aliases = ["/docs/grafana/latest/http_api/short_url/"]
+type = "docs"
+[menu.docs]
+name = "ShortURL"
+parent = "http_api"
++++
+
+# ShortURL API
```

Comment:
Add a short introduction that explains what this API is for. Remember, you will be explaining it to people who have never encountered it before. Tell them what it does, how to use it, what problems it solves, that sort of thing.